cli/crates/cli/tests/graphql/pagination/paginate-todo-lists.graphql

```diff
@@ -1,15 +1,21 @@
 query PaginateTodoLists($first: Int, $last: Int, $before: String, $after: String) {
-  todoListCollection(first: $first, last: $last, before: $before, after: $after) {
```

Comment:
Why remove the arguments here? Makes sense to leave it as a possibility

Reply:
Cursors aren't the same so I removed them. I kept `first` & `last` because they could eventually help find a bug and it's free even though it should be doing the same thing.
Otherwise adding specific cursors wouldn't bring that much IMHO. We're already testing root pagination. We could also test root pagination in conjunction with nested one, but that's not what we're doing currently. Should we add it eventually, we could just add the parameters then.